src/7-to-8/cheat-sheet.rst

```diff
@@ -177,6 +177,22 @@ View the parsed workflow configuration:
 
          cylc config <name/path>
 
+
+List configured platforms and platform groups
+---------------------------------------------
+
+.. seealso::
+
+   :ref:`ListingAvailablePlatforms`
+
+
+``cylc config --platform-names``
+   Print a list of configured platforms and platform groups.
+
+``cylc config --platform-meta``
+   Print out the full configuration of ``[platfroms]`` and
+   ``[platform groups]``
+
```

Comment:
This page is for Cylc 7 -> 8 CLI migration.

Reply:
I thought that wanting to know what platforms were available is exactly the sort of thing someone new to Cylc 8 from Cylc 7 might want a  quick reference to. But happy to remove if this is just for things which have a clearer mapping.

Reply:
The purpose of the migration guide is to outline the changes people must make to transition to Cylc 8. We need to keep it as minimal as possible and "signpost" into the main documentation. If we listed every helpful command this would end up pages long.